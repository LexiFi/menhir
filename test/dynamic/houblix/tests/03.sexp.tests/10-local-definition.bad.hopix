let main : int =
    let x : int = 1;
    y