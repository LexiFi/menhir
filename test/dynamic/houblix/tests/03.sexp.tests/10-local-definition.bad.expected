(((value
    (DefineValue
      (SimpleValue
        (((value (Id main))
           (position ((start_p ("" 1 0 4)) (end_p ("" 1 0 8)))))
          (((value
              (ForallTy ()
                ((value (TyCon (TCon int) ()))
                  (position ((start_p ("" 1 0 11)) (end_p ("" 1 0 14)))))))
             (position ((start_p ("" 1 0 10)) (end_p ("" 1 0 14))))))
          ((value
             (Define
               (SimpleValue
                 (((value (Id x))
                    (position ((start_p ("" 2 17 25)) (end_p ("" 2 17 26)))))
                   (((value
                       (ForallTy ()
                         ((value (TyCon (TCon int) ()))
                           (position
                             ((start_p ("" 2 17 29)) (end_p ("" 2 17 32)))))))
                      (position
                        ((start_p ("" 2 17 28)) (end_p ("" 2 17 32))))))
                   ((value
                      (Literal
                        ((value (LInt 1))
                          (position
                            ((start_p ("" 2 17 35)) (end_p ("" 2 17 36)))))))
                     (position ((start_p ("" 2 17 35)) (end_p ("" 2 17 36)))))))
               ((value
                  (Variable
                    ((value (Id y))
                      (position
                        ((start_p ("" 3 38 42)) (end_p ("" 3 38 43)))))
                    ()))
                 (position ((start_p ("" 3 38 42)) (end_p ("" 3 38 43)))))))
            (position ((start_p ("" 2 17 21)) (end_p ("" 3 38 43)))))))))
   (position ((start_p ("" 1 0 0)) (end_p ("" 3 38 43))))))
